manifests/charts/istio-operator/values.yaml

```diff
@@ -2,6 +2,7 @@ hub: gcr.io/istio-testing
 tag: latest
 operatorNamespace: istio-operator
 istioNamespace: istio-system
```

Comment:
I don't think you need to keep istioNamespace, it would be fine to leave a comment here and take it out.

Reply:
done.